packages/editor/playground/src/hooks/use-content.ts

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react"
+
+import updateURLParams from "../utils/update-url"
+
+const singleRow = `hello **strong**! hello *italic*! hello \`code\`! hello [link](https://www.google.com)!`
+const longContent = (singleRow.repeat(200) + "\n\n").repeat(5)
+
+const justCodeContent = `
+\`\`\`python
+while True:
+    print("hello world")
+\`\`\`
+`.trim()
+
+const defaultContent = [
+    `
+
+# Title
+
+hello world!
+
+`.trim(),
+    singleRow,
+    justCodeContent,
+    `
+
+- list item
+- list item
+  - [x] checked
+  - [ ] unchecked
+
+1. first
+1. second
+1. third
+
+`.trim(),
+].join("\n")
+export const contentMap: { [key: string]: string } = {
```

Comment:
Could we move `contentMap` to another file? E.g. `packages/editor/playground/src/content.ts`. 

It's a little weird that we import a non-hooks variable from `hooks/use-xxx.ts`